spec/system/subscriptions_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+describe "Subscriptions" do
+  let(:user) { create(:user, subscriptions_token: SecureRandom.base58(32)) }
+
+  context "Edit page" do
+    scenario "Render content in the user's preferred locale" do
+      user.update!(locale: "es")
+      visit edit_subscriptions_path(token: user.subscriptions_token)
+
+      expect(page).to have_content "Notificaciones"
+      expect(page).to have_content "Recibir un email cuando alguien comenta en mis propuestas o debates"
+      expect(page).to have_content "Recibir un email cuando alguien contesta a mis comentarios"
+      expect(page).to have_content "Recibir emails con información interesante sobre la web"
+      expect(page).to have_content "Recibir resumen de notificaciones sobre propuestas"
+      expect(page).to have_content "Recibir emails con mensajes privados"
+    end
+  end
+
+  context "Update" do
+    scenario "Allow update the status notification" do
+      user.update!(email_on_comment: true,
+                   email_on_comment_reply: true,
+                   email_on_direct_message: true,
+                   email_digest: true,
+                   newsletter: true)
```

Comment:
Maybe we could change a couple of them to `false`, and update just 4 of them, so we also check what happens when changing from `false` to `true` and what happens when the option isn't changed :thinking:.